crypto/src/hash/mds/mds_f64_8x8.rs

```diff
@@ -25,7 +25,6 @@ use math::{
 /// Hamish Ivey-Law and Jacqueline Nabaglo of Polygon Zero is based on Nabaglo's implementation
 /// in [Plonky2](https://github.com/mir-protocol/plonky2).
 /// The circulant matrix is identified by its first row: [23, 8, 13, 10, 7, 6, 21, 8].
-
```

Comment:
The doc comments are for the module - so, we should probably convert them into module level comments rather then attach them to the constant below.